source/config/__init__.py

```diff
@@ -983,7 +988,9 @@ def saveProfileTriggers(self):
 		This should be called whenever L{profilesToTriggers} is modified.
 		"""
 		if globalVars.appArgs.secure:
-			# Never save if running securely.
+			import ui
+			# Translators: Reported when an action cannot be performed because NVDA is in a secure screen
+			ui.message(_("Not available in secure context"))

```

Comment:
This method is called each time profile dialog is closed. Imagine that user looks at the available profiles in secure mode, closes the dialog, and gets the message that some action is not supported in secure screen. That would be rather confusing. I see no advantage to providing a message here.

Reply:
Should new profile dialogs be allowed to open? It seems like they can be opened but can't be saved in secure mode.

Reply:
I'm sorry - actually there is no way this method can be called when in secure mode as it is invoked when cancelling the triggers dialog which is disabled when running securely. My comment for the earlier methods in this file applies here as well - since this code is unreachable in secure mode there is no point in adding a message here.

Reply:
fixed in c928794